android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -242,6 +242,10 @@ private boolean shouldForceDefaultBrowserPrompt() {
         return !isDefaultBrowser();
     }
 
+    private boolean isNewOnboardingEnabled() {
```

Comment:
please use consistent naming to make things easier to search for

Reply:
this method has been removed as `unused`, but what was wrong with naming here?

Reply:
`NewOnboardingEnabled` vs `BRAVE_NEW_ANDROID_ONBOARDING` means that a search for either one will not match the other

Reply:
I'll resolve since this method was removed later, but just fyi for the future re naming